utilities/inference_utils.py

```diff
@@ -433,7 +434,14 @@ def run_inference(self, cmd: str) -> str:
         else:
             res, out, err = run_command(command=shlex.split(cmd), verify_stderr=False, check=False)
 
-        if not res:
+        if res:
+            if f"http/1.0 {HTTPStatus.SERVICE_UNAVAILABLE}" in out.lower():
```

Comment:
What if the error shows in stderr instead of stdout?

Reply:
res will be `False`
for this specific error, res is `True` and it appears in `out`